.github/ISSUE_TEMPLATE/improve-existing-docs.yaml

```diff
@@ -0,0 +1,47 @@
+name: Improve existing content
+description: Make a suggestion to improve the content in an existing article
+title: "['Improve existing content']: "
+labels:
+  - content
+body:
+  # HUBBERS BEWARE! THE GITHUB/DOCS REPO IS PUBLIC TO THE ENTIRE INTERNET. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.
```

Comment:
Should this be a part of the Markdown section below? I feel like since nobody will be looking at the YAML file except those of us maintaining it, we should either include it in the Markdown section or simply remove it.

Reply:
I added as a Markdown. Also changed the wording a little, let me know if it looks alright!